<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>874</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>AbilityTreeViewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/LOGO-AT.ico</normaloff>../res/LOGO-AT.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget{background-color:rgb(71,78,94);color:white;}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_title" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>48</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>48</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1" rowspan="2">
        <widget class="QLabel" name="label_113">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{border-image:url(../res/LOGO-ATW.png) 4 4 4 4 stretch stretch;background-color:rgb(0,0,0)}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QWidget" name="widget_line" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: qlineargradient(x1:0,y1:1,x2:1,y2:1,stop:0 #f4716e, stop:0.7 #42f875, stop:0.8 #1edffc);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QWidget" name="widget_title_right" native="true">
         <property name="styleSheet">
          <string notr="true">#widget_title_right{background: qlineargradient(x1:0,y1:1,x2:1,y2:1,stop:0 #000000, stop:0.7 #474e5e);}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_help">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#toolButton_help{border-image:url(./res/help.png);}#toolButton_help:hover{border-image:url(./res/help_hover.png);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_min">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#toolButton_min{border-image:url(./res/min.png);}#toolButton_min:hover{border-image:url(./res/min_hover.png);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_max">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#toolButton_max{border-image:url(./res/max.png);}#toolButton_max:hover{border-image:url(./res/max_hover.png);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_close">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#toolButton_close{border-image:url(./res/close.png);}#toolButton_close:hover{border-image:url(./res/close_hover.png);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QWidget" name="widget_title_left" native="true">
         <property name="styleSheet">
          <string notr="true">background: qlineargradient(x1:0,y1:1,x2:1,y2:1,stop:0.3 #474e5e, stop:1 #000000);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="leftMargin">
           <number>12</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>12</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>408</width>
              <height>11</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_center" native="true"/>
    </item>
   </layout>
  </widget>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actioncalib">
   <property name="text">
    <string>calib</string>
   </property>
  </action>
  <action name="actionparam">
   <property name="text">
    <string>param</string>
   </property>
  </action>
  <action name="actionconfig">
   <property name="text">
    <string>config</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
